.gitattributes

```diff
@@ -0,0 +1,6 @@
+#
```

Comment:
Let's not add the `.gitattributes` file if it's not necessary. (we generally don't add this to repos.)

Reply:
Removed https://github.com/ballerina-platform/module-ballerina-observe/pull/1/files#diff-54c2a1c9d62367d38139ae3d5d8200e06bc932ab7905f7e666d2c895382b2b81